core/uploader.go

```diff
@@ -147,7 +147,7 @@ func extractThumb(session drivers.OSSession, filename string, segment []byte) er
 		outFile,
 	}
 
-	timeout, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	timeout, cancel := context.WithTimeout(context.Background(), 8*time.Second)
```

Comment:
Instead of `bytes.Buffer` we might need to use `cmd.StdoutPipe()` and `cmd.StderrPipe()` https://stackoverflow.com/questions/35994907/go-combining-cmd-stdoutpipe-and-cmd-stderrpipe instead

Reply:
or `cmd.CombinedOutput()`